packages/table/src/spectrum-table.css

```diff
@@ -1199,7 +1199,7 @@ governing permissions and limitations under the License.
     ); /* [dir=ltr] .spectrum-Table--compact .spectrum-Table:not(.spectrum-Table--quiet) tbody.spectrum-Table-body .spectrum-Table-row:first-child .spectrum-Table-cell:last-child */
 }
 :host([dir='rtl'][density='compact'])
-    .spectrum-Table:not(.spectrum-Table--quiet)
+    .spectrum-Table:not([quiet])
```

Comment:
This being applied as `.spectrum-Table:not([quiet])` instead of being added as an attribute to the host element is certainly interesting. 

@Westbrook do you think this would require some kind of complex selector syntax in the config to handle this? This looks odd to me. 🤔

Reply:
Here is the CSS quiet table:
<img width="1267" alt="image" src="https://user-images.githubusercontent.com/1207520/225469681-18f83b3a-a519-47ba-8b5f-d5f302d658fd.png">
Not sure why this style is missing in SWC quiet table:
```
.spectrum-Table--quiet .spectrum-Table-body {
    border-width: 1px 0;
    background-color: var(--spectrum-table-m-regular-quiet-cell-background-color, var(--spectrum-alias-background-color-transparent));
}
```